rfcs/InputUnion.md

```diff
@@ -467,52 +444,8 @@ type Mutation {
 
 ##### [Doesn't inhibit schema evolution](#c-doesnt-inhibit-schema-evolution)
 
-Adding a nullable field to an input object could change the detected type of
```

Comment:
That's removing useful information.

Reply:
The example was very verbose, all other Evaluations are a single line. I'm concerned about this document blowing out in size / understandability

I'm going to use the Github markdown collapse section to collapse it by default. Also I'll re-phrase it using the Animal example we use in the rest of the doc